apps/src/templates/teacherDashboard/SectionRow.jsx

```diff
@@ -0,0 +1,133 @@
+import React, { Component, PropTypes } from 'react';
+import i18n from '@cdo/locale';
+import ProgressButton from '@cdo/apps/templates/progress/ProgressButton';
+
+export const sectionShape = PropTypes.shape({
+  id: PropTypes.number.isRequired,
+  name: PropTypes.string.isRequired,
+  loginType: PropTypes.oneOf(['word', 'email', 'picture']).isRequired,
```

Comment:
This is the only place this is used (at least on the client) so far. If it ends up being used elsewhere, I will consider an enum.